gsy_framework/enums.py

```diff
@@ -50,19 +50,34 @@ class AvailableMarketTypes(Enum):
     YEAR_FORWARD = 7
     INTRADAY = 8
 
+    def __str__(self):
+        return {
+            self.SPOT: "SPOT",
+            self.BALANCING: "BALANCING",
+            self.SETTLEMENT: "SETTLEMENT",
+            self.FUTURE: "FUTURE",
+            self.DAY_FORWARD: "DAY_FORWARD",
+            self.WEEK_FORWARD: "WEEK_FORWARD",
+            self.MONTH_FORWARD: "MONTH_FORWARD",
+            self.YEAR_FORWARD: "YEAR_FORWARD",
+            self.INTRADAY: "INTRADAY"
+        }[self]
```

Comment:
Is there a reason why you don't want to use `.name`? For example:
```py
>>> market_type = AvailableMarketTypes(3)
>>> market_type.name
# FUTURE
```

Reply:
Another note: I think that we have some naming issue here: we're mixing market types and product types. It would be nice to pick just one of the two terms (or decide what's the difference between one and the other). @spyrostz FYI